src/screens/Login/index.tsx

```diff
@@ -12,13 +12,25 @@ import styles from './styles';
 
 const Logout: FC = () => {
   const navigation = useNavigation();
-  const [email, setEmail] = useState('');
-  const [password, setPassword] = useState('');
+  const [email, setEmail] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+  const [emailError, setEmailError] = useState<string | null>(null);
+  const [passwordError, setPasswordError] = useState<string | null>('');
   const {actions} = useContext(AuthContext);
   const route = useRoute();
 
   async function submit() {
```

Comment:
idk. But maybe it will be better to use object for errors?
Like this:
```js
async function submit() {
    setErrors({});

    const errors = {
      ...(!email && { email: "Email is required" }),
      ...(!password && { password: "Password is required" })
    };

    if (!Object.keys(errors).length) {
      await actions.logIn(email, password);
      navigation.navigate(route.params?.routeName, {...route.params});
    } else {
      setErrors(errors)
    }
  }
```

Reply:
Sounds reasonable, thanks